components/Card.js

```diff
@@ -6,7 +7,7 @@ function Card({ backgroundImageSource, children }) {
     <div className="card shadow-xl image-full">
       {backgroundImageSource && (
         <figure>
-          <img src={backgroundImageSource} />
+          <Image alt="" src={backgroundImageSource} />
```

Comment:
I like your idea of exposing the alt text as a property of the component, perhaps something like `backgroundImageAltText`?

I know it's way beyond the scope of #103, haha...I'll accept this PR and file a separate issue for it.

Reply:
Yep my thoughts exactly. Bit beyond my React knowledge too, currently.